playbooks/install-vddk.yml

```diff
@@ -0,0 +1,38 @@
+---
+- name: CFME | Install VDDK
+  hosts: all
+  gather_facts: True
+  tasks:
+    - name: CFME | Install VDDK | Extract VDDK
+      unarchive:
+        src: "{{ cfme_vddk_path }}"
+        dest: /tmp/
```

Comment:
For robustness, I'd use the `tempfile` module  to create a temporary directory, then extract there.

```
- name: CFME | Install VDDK | create temp directory
  tempfile:
    state: directory
    prefix: vmware
  register: tmp_dir

- name: CFME | Install VDDK | Extract VDDK
  unarchive:
    src: "{{ cfme_vddk_path }}"
    dest: "{{ tmp_dir['path'] }}"

```

Reply:
Just updated to include this change.